core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -218,7 +218,7 @@ THE SOFTWARE.
           </j:if>
         </div>
       </j:if>
-      <div id="main-panel">
+      <div id="main-panel" role="main">
```

Comment:
Is there any `<main>` tag in use? Semantic HTML is better than using aria.

Reply:
Not yet, but that would be a better replacement here.